persistence-cassandra/core/src/main/resources/lagom-test-embedded-cassandra.yaml

```diff
@@ -25,6 +25,8 @@ saved_caches_directory: $DIR/saved_caches
 
 hints_directory: $DIR/hints
 
+cdc_raw_directory: $DIR/cdc_raw
+
```

Comment:
See discussion in PR #688
https://github.com/lagom/lagom/pull/688/files/86244a9188dad011402ec4eb63a2f34167e80322#r121856022